Assets/Mirror/Runtime/MessagePacker.cs

```diff
@@ -122,6 +122,14 @@ public static bool UnpackMessage(NetworkReader messageReader, out int msgType)
             T message = default;
             try
             {
+                if (requireAuthenication && !networkMessage.conn.isAuthenticated)
+                {
+                    // message requires authentication, but the connection was no authnticated
+                    Debug.LogError($"Closed connection: {networkMessage.conn.connectionId}. Received message {typeof(T)} that required authentication, but the user has not authenticated yet");
```

Comment:
```suggestion
                    Debug.LogWarning($"Closed connection: {networkMessage.conn.connectionId}. Received message {typeof(T)} that required authentication, but the user has not authenticated yet");
```
I don't think this is error-worthy here. attackers will try to do this eventually, it's almost expected